components/econet/__init__.py

```diff
@@ -18,6 +18,7 @@
 CONF_DATAPOINT_TYPE = "datapoint_type"
 CONF_REQUEST_MOD = "request_mod"
 CONF_REQUEST_ONCE = "request_once"
+CONF_REQUEST_INTERVAL = "request_mod_update_intervals"
```

Comment:
It seems like generally properties aren't pluralized. So I think it should be `request_mod_update_interval` rather tan `request_mod_update_intervals`.

Reply:
Do you have any examples of this for pluralized vs non for things like this? This is based on select > options, which is pluralized.

```
select:
  - platform: econet
    name: "Away Fan Mode"
    enum_datapoint: VACA_FAN
    entity_category: "config"
    options:
      0: "Automatic"
      1: "Speed 1 (Low)"
      2: "Speed 2 (Medium Low)"
      3: "Speed 3 (Medium)"
      4: "Speed 4 (Medium High)"
      5: "Speed 5 (High)"
```

Reply:
Generally I guess even
```
select:
```
and similar are not pluralized... maybe that's because those are "components"?

Reply:
substitutions:

packages:  # Change this file to the correct one(s) for your device

are both pluralized...

Reply:
Filter is pluralized too:
```
    filters:
      - delta: 0.9
```